src/resolve/resolved-varlink.c

```diff
@@ -37,6 +37,22 @@ static void lookup_parameters_destroy(LookupParameters *p) {
         free(p->name);
 }
 
+static int dns_query_new_for_varlink(
+                Manager *m,
+                DnsQuery **ret,
+                DnsQuestion *question_utf8,
+                DnsQuestion *question_idna,
+                DnsPacket *question_bypass,
+                sd_varlink *link,
+                int ifindex,
+                uint64_t flags) {
```

Comment:
Please make the ordering consistent with `dns_query_new_for_dbus()`, that is, move `sd_varlink *link` to the last.